### ISSUE TITLE
Fabric 1.21 port Please.

### ISSUE THREAD

**Issue by MotMomet** [blockninja124/ResourceControlMod#1]

**Comment by blockninja124** [comment]
I could probably port this to fabric pretty easily, since I'm not really using any forge-based code (just brigadier mainly). The only thing is I would have to look into whether 1.21 with all the component changes and stuff would significantly affect my code?

Basically, fabric might be coming soon-ish, and I'll take a look at getting it to 1.21 (that is, if you meant 1.21 and didn't just mistype 1.20.1)

**Comment by blockninja124** [comment]
Although honestly, you could probably run this mod through [sinytra connector](https://modrinth.com/mod/connector) without many issues. But if you do mean 1.21, then yeah I will have to look into that.

**Comment by MotMomet** [comment]
> I could probably port this to fabric pretty easily, since I'm not really using any forge-based code (just brigadier mainly). The only thing is I would have to look into whether 1.21 with all the component changes and stuff would significantly affect my code?
> 
> Basically, fabric might be coming soon-ish, and I'll take a look at getting it to 1.21 (that is, if you meant 1.21 and didn't just mistype 1.20.1)

Yeah, I meant the Fabric 1.21 version. I would really appreciate it.

**Comment by MotMomet** [comment]
are you going to port 1.21 fabric btw? like soon? or is it just planned?

**Comment by blockninja124** [comment]
Mostly just planned right now, since I’m currently busy working on another mod. Once I have a bit more free time I’ll do it, hopefully soon.

**Comment by MotMomet** [comment]
> Mostly just planned right now, since I’m currently busy working on another mod. Once I have a bit more free time I’ll do it, hopefully soon.

ok, cool!

**Comment by blockninja124** [comment]
I've run into some issues while porting to 1.21, and apperently RC doesn't work on servers... so for now I'm putting off the 1.21 port until I've switched to a network/packet based system to hopefully fix that issue.

**Comment by blockninja124** [comment]
I fixed RC on servers in version 2.0.0, and as of the new version 3.0.0 it’s on 1.21! Neoforge and fabric.
